### ISSUE TITLE
ctrl-alt-a/e not working

### ISSUE THREAD

**Issue by green-stalk** [zombieFox/nightTab#232]
I'm fairly new to all of this complicated computer stuff, so it is somewhat likely that I've just installed something wrong, but I'm unable to use those two keyboard shortcuts in nightTab. ctrl-alt-a does nothing outside a text box, but when I'm in a text box it places the á character, and the same for é. 
I think this is what's preventing me from using those shortcuts normally, but then surely other people would have the same problem because I haven't added any custom shortcuts or anything.
ctrl-alt-g and ctrl-alt-d work fine, and they don't place any special characters, so I'm sure it can't be a coincidence that those are the ones that work.
I'm not 100% sure this is the right place to ask, but can anyone help with this, and is it even anything to do with á/é at all?

**Comment by zombieFox** [comment]
Hi,

I'll try to help. Can you provide a few more details please?
What browser (and the version) are you using?
What OS (and the version) are you using?
Do you have any other extensions installed which have anything to do with the keyboard?
Have you customised any OS level keyboard shortcuts?

The shortcuts <kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>a</kbd> and <kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>e</kbd> have been working as expected in my tests and since the last release.

**Comment by green-stalk** [comment]
- 86.0
- Windows 10 version 1909
- No keyboard related extensions installed, they're mostly privacy-related, with this and tree-style tabs for the ui
- I've never changed or added any keyboard shortcuts

It's weird because it can't be a coincidence that the two shortcuts that aren't working have another function, but then since I haven't changed anything with shortcuts they must be the default, so why aren't more people having this problem?

**Comment by zombieFox** [comment]
Agreed, no one else has reported this issue before.

Can you check to see if any of the other extensions have keyboard shortcuts?

**Comment by green-stalk** [comment]
I have since removed all extensions other than this and tree tabs, and checked the demo page, and the shortcuts still don't work as intended. Before I removed the extensions I tried adding a custom ctrl+alt+a/e shortcut, and none of them would even let me, which is a bit weird. Everytime I typed them in the text vanished without any error message or anything, so I guess for some reason my laptop has decided to not allow anyone to use those shortcuts apart from for á/é

**Comment by zombieFox** [comment]
Very odd.

The code that [controls the keyboard shortcuts](https://github.com/zombieFox/nightTab/blob/4b6e1210dad5d258b6a871cac8a9d410ca73a2b5/src/js/keyboard.js#L54) all use the same pattern and logic. So if some keyboard shortcuts work but others don't suggests to me that something on your laptop is prevent that combination of keys from being used. This sounds like something outside the control of nightTab.

**Comment by green-stalk** [comment]
yeah it seems like a me issue, so unless there's a way to change those default shortcuts, I'll resign myself to having to click the add/edit buttons like some kind of chump lol

Thanks for the help :)
